shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -158,7 +159,8 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
         }
         result.setBoundsInScreen(bounds);
         result.setVisibleToUser(true);
-        result.setEnabled(!object.hasFlag(Flag.IS_DISABLED));
+        result.setEnabled(!object.hasFlag(Flag.HAS_ENABLED_STATE) ||
```

Comment:
Does it make sense to call the setEnabled method on the result at all if the object has no `HAS_ENABLED_FLAG`?

Reply:
I was experimenting with that. However, Android doesn't really seem to differentiate between "enabled" and "doesn't have enabled state". Therefore, not calling setEnabled at all yields strange behavior.

Reply:
Android doesn't have the concept of "enablable", and things that can't be enabled are therefore enabled on Android.

iOS doen't have the concept of "enablable" either, and things that can't be enabled therefore are similarly considered "not disabled", i.e. enabled, hence the equivalent (but reversed) code below (reversed because Android asks whether something is enabled but iOS asks whether it's disabled).